src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -662,7 +669,9 @@ def _log_request(request):
         logger.info("Request headers:")
         for header, value in request.headers.items():
             if header.lower() == 'authorization':
-                value = value[:15] + '*****'
+                # Trim at least half of the token but keep at most 20 characters
+                preserve_length = min(int(len(value) * 0.5), 20)
+                value = value[:preserve_length] + '*****'
```

Comment:
Do we need to consider whether the number of characters of `value` should remain the same after the last few bits are encrypted by `*`? Is the length of `value` in the previous logic unchanged after encryption?

Reply:
I don't think so, otherwise a Bearer token is too long.

Reply:
Personally, if `*****` does not contain information about the number of characters, perhaps `...` is more appropriate to indicate omission.
For example: `'Authorization': 'Bearer eyJ0eXAiOiJKV...'`
It's just my personal preference, what do you think about it~

Reply:
Nice idea. 😃 Changed as requested.